### ISSUE TITLE
Verifying progress text

### ISSUE THREAD

**Issue by SiIky** [equeim/tremotesf-android#36]
On the torrents list, while a torrent is being verified, the progress text shows `0.n%` (e.g. `0.3%` meaning `30%`) which is wrong and very confusing. I suggest showing either an actual percentage (0\~100), or decimal values (0\~1) like now, but without the percentage sign.

I think the code for this is here:

https://github.com/equeim/tremotesf-android/blob/d25a54cd5addeae66e4443617795fa3ba15decd5/app/src/main/kotlin/org/equeim/tremotesf/TorrentData.kt#L56-L57

I can't find where the percentage sign is added, but I found that `recheckProgress` is a `double` (0\~1 I think?):

https://github.com/equeim/tremotesf-android/blob/d25a54cd5addeae66e4443617795fa3ba15decd5/libtremotesf/jni/java/org/equeim/libtremotesf/Torrent.java#L87-L89

I know absolutely no Kotlin, so sorry if this is wrong, but does this come close to being ok?

```patch
diff --git a/app/src/main/kotlin/org/equeim/tremotesf/TorrentData.kt b/app/src/main/kotlin/org/equeim/tremotesf/TorrentData.kt
index ae2a2d2..27e0d2f 100644
--- a/app/src/main/kotlin/org/equeim/tremotesf/TorrentData.kt
+++ b/app/src/main/kotlin/org/equeim/tremotesf/TorrentData.kt
@@ -54,7 +54,7 @@ class TorrentData(val id: Int, val torrent: Torrent, private val context: Contex
                 Torrent.Status.QueuedForDownloading,
                 Torrent.Status.QueuedForSeeding -> context.getString(R.string.torrent_queued)
                 Torrent.Status.Checking -> context.getString(R.string.torrent_checking,
-                        DecimalFormat("0.#").format(recheckProgress))
+                        DecimalFormat("#").format(recheckProgress))
                 Torrent.Status.QueuedForChecking -> context.getString(R.string.torrent_queued_for_checking)
                 Torrent.Status.Errored -> errorString
                 else -> ""
@@ -71,7 +71,7 @@ class TorrentData(val id: Int, val torrent: Torrent, private val context: Contex
         private set
     var isFinished = torrent.isFinished
         private set
-    var recheckProgress = torrent.recheckProgress()
+    var recheckProgress = torrent.recheckProgress() * 100
         private set
     var eta = torrent.eta()
         private set
@@ -122,7 +122,7 @@ class TorrentData(val id: Int, val torrent: Torrent, private val context: Contex
             sizeWhenDone = torrent.sizeWhenDone()
             percentDone = torrent.percentDone()
             isFinished = torrent.isFinished
-            recheckProgress = torrent.recheckProgress()
+            recheckProgress = torrent.recheckProgress() * 100
             eta = torrent.eta()
             downloadSpeed = torrent.downloadSpeed()
             uploadSpeed = torrent.uploadSpeed()
```

Tell me what you think! :)

**Comment by Sak94664** [comment]
I think this adjustment is nice :)
But if you pay more attention to transmission clients you will notice many other clients have this same problem(transmission-qt and transmission-gtk).And if I am correct ,the webui has the same bug as well.
It's not tremotesf's fault but maybe it's the upstream's fault.(I only checked thw software on Debian9 and Transmission 2.92 webui,not sure whether they have already fixed it)

**Comment by SiIky** [comment]
Thanks for your input, I wasn't aware of that! The only clients I use are `transmission-remote` (the CLI client; doesn't have this problem), and `tremotesf` on Android.

I went looking for how `transmission-remote` does it just now, and found this:

https://github.com/transmission/transmission/blob/44fc571a6737e98d04c09e479fd6682222abc13d/utils/remote.c#L854-L863

Git Blame says that line hasn't been changed in 3 years. The `rpc-spec.txt` isn't explicit about what `recheckProgress` really is:

https://github.com/transmission/transmission/blob/44fc571a6737e98d04c09e479fd6682222abc13d/extras/rpc-spec.txt#L229

So I agree it could be improved; however, there's this that I also just found:

https://github.com/transmission/transmission/blob/44fc571a6737e98d04c09e479fd6682222abc13d/libtransmission/transmission.h#L1705-L1710
